Demo/Sources/Model/MediaDescription.swift

```diff
@@ -39,4 +46,16 @@ enum MediaDescription {
     static func style(for media: SRGMedia) -> Style {
         media.timeAvailability(at: Date()) == .available ? .standard : .disabled
     }
+
+    private static func date(for media: SRGMedia) -> String {
+        dateFormatter.string(from: media.date)
+    }
+
+    private static func duration(for media: SRGMedia) -> String {
+        minuteFormatter.string(from: max(media.duration / 1000, 60))!
```

Comment:
I understand this fallback to prevent displaying something like **0 min**.
In this context, is it the application's role to hide invalid or incorrect data? 🤷‍♂️

Reply:
This is not really invalid, this is the convention we decided to apply in the Play app (items less than 1 min are displayed as _1 min_). I just reused the same idea :)